### DISCUSSION TITLE
不蒜子在手机和平板等移动端统计不正常。

### DISCUSSION THREAD

**Discussion by leimao** [ppoffice/hexo-theme-icarus#996] [Q&A]
大家好，我在Icarus主题里使用了不蒜子统计后，在电脑上的数据显示是正常的。
但是如果我使用iPhone手机或者iPad平板上任意的浏览器来看网站，数据是错误的。而且感觉这个数据是整个网站的点击总量。我不清楚发生了什么事情，请懂的朋友指教一下。谢谢。
我的网站是：
https://leimao.github.io/

**Comment by ppoffice** [comment]
@leimao 你浏览不蒜子官网，数据会正常显示么？

**Reply by leimao** [reply]
@ppoffice 数据在平板和电脑看来都是一致的。事实上我之前看过一些Icarus用户的网站，他们的不蒜子统计数据在移动端和电脑都是一致的。就我不一致。

**Comment by leimao** [comment]
感谢作者 @ppoffice
这个是[不蒜子官网](https://busuanzi.ibruce.info/)，我怎么从官网查看文章的统计数据？

**Comment by leimao** [comment]
@ppoffice 不好意思，我明白了。我这就用手机和电脑分别看一下不蒜子官网。

**Comment by leimao** [comment]
@ppoffice 今天不知道怎么的我能够在其他Icarus主题的网站重复类似的现象。
比如我们访问如下网址：
https://removeif.github.io/java/frame/springboot-dubbo-nacos-example.html

PC FireFox： 300多次

![image](https://user-images.githubusercontent.com/17606112/147210496-fb532ae0-4e93-4023-8568-526311515678.png)

PC Chrome： 300多次

![image](https://user-images.githubusercontent.com/17606112/147211427-124a26c9-1f74-430b-95a5-392314ed0e58.png)

iPad Chrome： 6万多次

![IMG-0064](https://user-images.githubusercontent.com/17606112/147211517-ab64d270-c8ef-4726-9eff-219e153638db.PNG)

**Reply by ppoffice** [reply]
可能和iPad OS的webkit内核有关。下面这篇文章有一些措施你可以试一下

https://removeif.github.io/theme/%E4%B8%8D%E8%92%9C%E5%AD%90%E7%BB%9F%E8%AE%A1%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98.html

另外你可以尝试仅用其他的一些插件，特别是谷歌的Adsense来观察是否对不蒜子有影响。

**Comment by zhenda-hub** [comment]
我也发现有这个问题
